spec/amp-cache-transform.md

```diff
@@ -163,12 +201,17 @@ the latter response.
 If the proxy can ensure that a cached response satisfies a new request, then it
 can serve that response. It can do that by comparing the `AMP-Cache-Transform`
 response header of the cached response to the `AMP-Cache-Transform` request
-header of the new request:
-
- 1. If the request includes an unparameterised `any` identifier and the response
-    includes a `AMP-Cache-Transform` header, then it satisfies the request.
- 2. If the response identifier is included in the request list, and
-    unparameterised in both cases, then it satisfies the request.
+header of the new request. For instance, the following algorithm determines if a
+response satisfies a request:
+
+ 1. If the response matches any parameterised identifier in the request list,
```

Comment:
No need to implement but just a suggestion.

Since these are about response either matching or not matching, could move that to the beginning for each one

1. The response doesn't match if it's identifier is not any...
2. The response doesn't match if it includes a v parameter...
3. The response doesn't match if it includes any other...
4. The response matches otherwise.

Reply:
Done and then undone as I reworked this section to fix @Gregable's pronoun problem.